docs/source/app_developers_guide/kubernetes.rst

```diff
@@ -0,0 +1,677 @@
+           [2018-08-16 19:12:51.198 INFO     processor_handlers] registered transaction processor: connection_id=084ecc34848d7293821a3f2c58adc4f703572a368783afd901004bfd982e82ce5fe6e1f6e6e08de9fe6fc25c98ae20e55fa493f4f510824a2bb4a5fe00210c81, family=sawtooth_settings, version=1.0, namespaces=['000000']
+           [2018-08-16 19:12:51.235 DEBUG    genesis] Produced state hash 0e682c25c3390a718ec560bb45d5180924f255210d9d4521eaac019800603731 for genesis block.
+           [2018-08-16 19:12:51.238 INFO     genesis] Genesis block created: 20d7b6657721758d1ad1a3392daadd57473d84e1e1c8c58c14ec862ff7fbf44a3bef4d82c40052dd8fc2808191f830447df59fe074aea02a000ff64bc458e256 (block_num:0, state:0e682c25c3390a718ec560bb45d5180924f255210d9d4521eaac019800603731, previous_block_id:0000000000000000)
+           [2018-08-16 19:12:51.238 DEBUG    chain_id_manager] writing block chain id
+           [2018-08-16 19:12:51.239 DEBUG    genesis] Deleting genesis data.
+           [2018-08-16 19:12:51.239 DEBUG    selector_events] Using selector: ZMQSelector
+           [2018-08-16 19:12:51.240 INFO     interconnect] Listening on tcp://eth0:8800
+           [2018-08-16 19:12:51.241 DEBUG    dispatch] Added send_message function for connection ServerThread
+           [2018-08-16 19:12:51.241 DEBUG    dispatch] Added send_last_message function for connection ServerThread
+           [2018-08-16 19:12:51.243 DEBUG    gossip] Number of peers (0) below minimum peer threshold (3). Doing topology search.
+           [2018-08-16 19:12:51.244 INFO     chain] Chain controller initialized with chain head: 20d7b6657721758d1ad1a3392daadd57473d84e1e1c8c58c14ec862ff7fbf44a3bef4d82c40052dd8fc2808191f830447df59fe074aea02a000ff64bc458e256 (block_num:0, state:0e682c25c3390a718ec560bb45d5180924f255210d9d4521eaac019800603731, previous_block_id:0000000000000000)
+           [2018-08-16 19:12:51.244 INFO     publisher] Now building on top of block: 20d7b6657721758d1ad1a3392daadd57473d84e1e1c8c58c14ec862ff7fbf44a3bef4d82c40052dd8fc2808191f830447df59fe074aea02a000ff64bc458e256 (block_num:0, state:0e682c25c3390a718ec560bb45d5180924f255210d9d4521eaac019800603731, previous_block_id:0000000000000000)
+
+You can also access a component's log messages by connecting to the container
+and examining the local log files. In each container, the Sawtooth log files for
+that component are stored in the directory ``/var/log/sawtooth``. Each component
+(validator, REST API, and transaction processors) has both a debug log and an
+error log.
+
+For example, you can connect to the validator container and display the contents
+of ``/var/log/sawtooth``:
+
+.. code-block:: console
+
+   $ kubectl exec -it $(kubectl get pods | awk '/sawtooth-0/{print $1}') --container sawtooth-validator -- bash
+   root@podname#
```

Comment:
I like how we have the ls command here in the next example. What do you think of putting that here to? as is it looks like the logs would be in /